Library/src/Multiplayer/EventSerialisation.cpp

```diff
@@ -28,38 +28,38 @@ namespace
 {
 ESequenceUpdateType ESequenceUpdateIntToUpdateType(uint64_t UpdateType)
 {
-    ESequenceUpdateType SequenceUpdateType = ESequenceUpdateType::Invalid;
-
-    switch(UpdateType)
-    {
-    	case 0:
-    	{
-            SequenceUpdateType = ESequenceUpdateType::Create;
-    		break;
-    	}
-        case 1:
-    	{
-            SequenceUpdateType = ESequenceUpdateType::Update;
-    		break;
-    	}
-        case 2:
-    	{
-            SequenceUpdateType = ESequenceUpdateType::Rename;
-    		break;
-    	}
-        case 3:
-    	{
-            SequenceUpdateType = ESequenceUpdateType::Delete;
-    		break;
-    	}
-    	default:
-    	{
-            CSP_LOG_ERROR_MSG("SequenceChangedEvent - Detected an unsupported update type.");
-    		break;
-    	}
-    }
-
-    return SequenceUpdateType;
```

Comment:
this should be spaces not tab, need to check my vs2022 in case the setting have been missed from setup